components/Breadcrumb/src/index.tsx

```diff
@@ -108,19 +72,26 @@ export const Breadcrumb: React.FC<BreadcrumbProps> = (props: BreadcrumbProps) =>
         ) : (
           <BreadcrumbItem item={item} isLastItem={isLastItem} contentNumber={contentNumber} />
         )}
-      </li>
+        <meta content={contentNumber.toString()} itemProp="position" />
+      </BreadcrumbListItem>
     );
   });
 
   return (
-    <nav aria-label="Breadcrumb" className="denhaag-breadcrumb">
+    <BreadcrumbNavigation aria-label="Breadcrumb">
       <ResponsiveContent>
-        <ol className="denhaag-breadcrumb__list" itemScope itemType="https://schema.org/BreadcrumbList">
+        <BreadcrumbList itemScope itemType="https://schema.org/BreadcrumbList">
```

Comment:
I think `itemScope={itemScope}` is needed here? now it's always going to be true